completed/basics_complete.py

```diff
@@ -0,0 +1,202 @@
+"""
+Strings
+"""
+# create a string and assign it to a variable
+my_string = "We're going to learn Python at #NICAR16"
+
+# print your string. If you're using the interactive interpreter
+# (rather than running this script from a saved file), you don't need to type "print"
+print my_string
+
+# get the type
+type(my_string)
+
+# get the length
+len(my_string)
+
+# remember that a string is just a series of characters, not a word or a sentence
+# so you can "slice" off pieces of a string based on the index of the characters in it.
+# Try to slice off the last 5 characters in my_string
+my_string[-5:]
+
+# convert it to lowercase
+my_string.lower()
+
+# convert it to uppercase
+my_string.upper()
+
+# convert it to titlecase
+my_string.title()
+
+# concatenate strings
+my_string + my_string
+
+# split strings
+my_string.split("learn")
+
+# join a list of strings
+"+++".join([my_string, my_string])
+
+# remove a character
+my_string.replace(" ", "##")
+
+# strip whitespace
+"  this string has whitespace  ".strip()
+
+# removes only leading whitespace chars
+"  this string has whitespace  ".lstrip()
+
+# removes only trailing whitespace chars
+"  this string has whitespace  ".rstrip()
+
+# I need help remembering what methods my variable has available
+dir(my_string)
+
+# I need help on a specific method
+help(my_string.lower)
+
+
+"""
+integers
+"""
+# create an integer
+my_integer = 25
+
+# print your integer
+print my_integer
+
+# get the type of your integer
+type(my_integer)
+
+# do some addition
+my_sum = my_integer + 10
+print my_sum
+
+# do some subtraction
+my_difference = my_integer - 10
+my_difference
+
+# do some multiplication
+my_product = my_integer * 10
+my_product
+
+# do some division
+my_dividend = my_integer / 10
+my_dividend
```

Comment:
The result of a division problem is called the quotient. The dividend in this case is my_integer, with 10 being the divisor.